README.md

```diff
@@ -57,13 +57,13 @@ Function (and decorator) that converts a React component definition, React compo
 
 #### Functional Components
 
-`React.memo` is automatically applied to functional components provided to `observer`. `Observer` does not accept a functional component already wrapped in `React.memo`, or an `observer`, in order to avoid consequences that might arise as a result of wrapping it twice.
+`React.memo` is automatically applied to functional components provided to `observer`. `observer` does not accept a functional component already wrapped in `React.memo`, or an `observer`, in order to avoid consequences that might arise as a result of wrapping it twice.
```

Comment:
Just for readability, too many observers imo :)
```suggestion
`React.memo` is automatically applied to functional components provided to `observer`. It does not accept a functional component already wrapped in `React.memo`, or an `observer`, in order to avoid consequences that might arise as a result of wrapping it twice.
```

Reply:
Yeah, I thought about that.  Using 'it' instead makes it seem like you're talking about React.memo as it is the subject of the previous sentence.  And, rearranging the sentence could add confusion.  It's redundant, but it's clear, which is what I was aiming for.